src/screens/steem-connect/hiveSigner.js

```diff
@@ -51,12 +51,13 @@ class HiveSigner extends PureComponent {
               dispatch(loginAction(true));
 
               if (isPinCodeOpen) {
-                dispatch(
-                  openPinCodeModal({
+                navigation.navigate({
+                  routeName: ROUTES.SCREENS.PINCODE,
```

Comment:
Here?

Reply:
pinCode is now a navigable screen instead of modal, that is why no longer need to dispatch